src/languages/twig.js

```diff
@@ -28,17 +29,17 @@ export default function(hljs) {
   var FILTER = {
     begin: /\|[A-Za-z_]+:?/,
     keywords:
-      'abs batch capitalize column convert_encoding date date_modify default ' +
-      'escape filter first format inky_to_html inline_css join json_encode keys last ' +
-      'length lower map markdown merge nl2br number_format raw reduce replace ' +
-      'reverse round slice sort spaceless split striptags title trim upper url_encode',
+      'abs batch capitalize column convert_encoding country_name currency_name currency_symbol data_uri date date_modify default ' +
+      'escape filter first format format_currency format_date format_datetime format_number format_time html_to_markdown inky_to_html inline_css join json_encode keys language_name last ' +
+      'length locale_name lower map markdown_to_html merge nl2br number_format raw reduce replace ' +
+      'reverse round slice sort spaceless split striptags timezone_name title trim u upper url_encode',
```

Comment:
Please add `|0` to score the `u` as 0 relevancy.

Reply:
I don't know if this is what you mean :)

Reply:
Ype!